pom.xml

```diff
@@ -31,6 +31,16 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.json/json -->
+		<!-- https://mvnrepository.com/artifact/org.json/json/20201115 -->
+		<!-- https://stackoverflow.com/a/5245881 -->
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20201115</version>
```

Comment:
This is used to parse JSON-strings so we can actually access their fields such as:

```java
JSONObject jsonObject = new JSONObject(body);
ref = jsonObject.getString("ref");
```
instead of manually parsing them using regexes.

Reply:
Regex would be a pain, I can feel it already